WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -37,6 +37,13 @@ class NoticePresenter: NSObject {
         let notice: Notice
         let containerView: NoticeContainerView?
     }
+    /// Used to determine if the keyboard is currently shown and what its height is.
+    private struct KeyboardPresentation {
```

Comment:
What do you think of an enum with two states:
```swift
case notPresent
case present(height: CGFloat)
```
?

Reply:
That's perfect! I'll add that.

Reply:
Done in e619cb746e8ce443fc32ad562a9c6276bdbcc8cd.